week03/script.js

```diff
@@ -0,0 +1,61 @@
+const humanScore = document.getElementById("human-score");
+const computerScore = document.getElementById("computer-score");
+
+const result = document.getElementById("result");
+
+const humanChoiceImg = document.getElementById("human-choice");
+const computerChoiceImg = document.getElementById("computer-choice");
+
+const rockBtn = document.getElementById("rock");
+const scissorsBtn = document.getElementById("scissors");
+const paperBtn = document.getElementById("paper");
+
+const resetBtn = document.getElementById("reset");
+
+// 가위바위보 이미지 매핑
+const choices = {
+  rock: "images/rock.png",
+  scissors: "images/scissors.png",
+  paper: "images/paper.png",
+};
+
+// 가위바위보 게임 실행
+function Game(humanChoice) {
+  humanChoiceImg.src = choices[humanChoice];
+
+  const computerKeys = Object.keys(choices);
+  const computerChoice = computerKeys[Math.floor(Math.random() * 3)]; // 랜덤 선택
+
+  computerChoiceImg.src = choices[computerChoice];
+
+  // 승패
+  if (humanChoice === computerChoice) {
+    result.innerText = "무승부O.O";
+  } else if (
+    (humanChoice === "rock" && computerChoice === "scissors") ||
+    (humanChoice === "scissors" && computerChoice === "paper") ||
+    (humanChoice === "paper" && computerChoice === "rock")
+  ) {
+    result.innerText = "승리!>.<";
+    const curNum = parseInt(humanScore.innerText) || 0;
+    humanScore.innerText = curNum + 1;
+  } else {
+    result.innerText = "패배ㅜ.ㅜ";
+    const curNum = parseInt(computerScore.innerText) || 0;
+    computerScore.innerText = curNum + 1;
+  }
+}
+
+// 가위바위보 버튼
+rockBtn.onclick = () => Game("rock");
+scissorsBtn.onclick = () => Game("scissors");
```

Comment:
```suggestion
rockBtn.addEventListener("click", () => Game("rock"));
scissorsBtn.addEventListener("click", () => Game("scissors"));
paperBtn.addEventListener("click", () => Game("paper"));
```
요렇게 `addEventListenter`를 이용해 주시면 보다 이벤트 핸들링에서 유지 보수도 쉽고,
나중에 `removeEventListener` 를 이용해서 event 핸들링을 없앨 수 있어서 추천드려요